main.go

```diff
@@ -328,6 +332,18 @@ func loadConfig(filename string) (*FileConfig, error) {
 	return c, nil
 }
 
+func loadEmailsConfig(filename string) (*EmailsConfig, error) {
```

Comment:
It looks like `filename` is ignored here (and also in `loadConfig`, where I copied this code from). Should we delete this argument? Or, I guess, just use `filename` instead of reading the global variable `emailsCfg`?

Reply:
Yeah this is a mistake, I meant to use the local variable in the function instead of the global.